ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -127,7 +127,7 @@ export class ImportCommand extends IronfishCommand {
   async importFile(path: string): Promise<AccountImport> {
     const resolved = this.sdk.fileSystem.resolve(path)
     const data = await this.sdk.fileSystem.readFile(resolved)
-    return this.stringToAccountImport(data)
```

Comment:
suggest adding a test for this case

Reply:
Seems like a good idea, once we unblock our users we can look into how to not only write tests, but make this code handle old schemas in a real way.